azure_arc_vsphere_jumpstart/resource_bridge/powershell/vCenter_onboarding.ps1

```diff
@@ -0,0 +1,250 @@
+
+log "Enabling long path support for python..."
+Start-Process powershell.exe -verb runas -ArgumentList "Set-ItemProperty -Path HKLM:\SYSTEM\CurrentControlSet\Control\FileSystem -Name LongPathsEnabled -Value 1" -Wait
+
+py -m venv .temp\.env
+
+
+log "Checking previous installations of Azure CLI"
+
+$AzureCLI = Get-WmiObject -Class Win32_Product  | Where-Object{$_.Name -eq "Microsoft Azure CLI"}
+
+If([string]::IsNullOrWhiteSpace($AzureCLI)) {            
+    log "No previous Azure CLI installation was found"
+} else {
+    log "Removing Azure CLI current install"
+    $AzureCLI.uninstall()
+}
+
+
+log "Installing 64 bit Azure CLI"
+log "This might take a while..."
+
+.temp\.env\Scripts\python.exe -m pip install --upgrade pip wheel setuptools >> $logFile
+.temp\.env\Scripts\pip install azure-cli >> $logFile
+
+.temp\.env\Scripts\Activate.ps1
+
+
+try {
+
+    az provider register -n Microsoft.ResourceConnector --wait
+    az provider register -n Microsoft.ConnectedVMwarevSphere --wait
+
+    log "Installing az cli extensions for Arc"
+    az extension add --name arcappliance
+    az extension add --name k8s-extension
+    az extension add --name customlocation
+    az extension add --name connectedvmware
+
+    log "Logging into azure"
+
+    az login --service-principal -u $spnClientId -p $spnClientSecret --tenant $spnTenantId
+
+    az account set -s $subscriptionId
+    if ($LASTEXITCODE) {
+        $Error[0] | Out-String >> $logFile
+        throw "The default subscription for the az cli context could not be set."
+    }
+
+    log "Step 1/5: Workstation was set up successfully"
+
+
+    log "Step 2/5: Creating the Arc resource bridge"
+    log "Provide vCenter details to deploy Arc resource bridge VM. The credentials will be used by Arc resource bridge to update and scale itself."
+    $user_in = ""
+    foreach ($val in $loginValues) { $user_in = $user_in + "`n" + $val }
+    $login_password = $loginValues[2]
+
+    log "Validating"
+    $user_in | az arcappliance validate vmware --config-file .\arcbridge-appliance-stage.yaml 2>> $logFile
+    log "Preparing"
+    $user_in | az arcappliance prepare vmware --config-file .\arcbridge-appliance-stage.yaml 2>> $logFile
+    log "Deploying"
+    $user_in | az arcappliance deploy vmware --config-file .\arcbridge-appliance-stage.yaml 2>> $logFile
+    sleep -Seconds 20
+    log "Creating"
+    $user_in | az arcappliance create vmware --config-file .\arcbridge-appliance-stage.yaml --kubeconfig .\kubeconfig 2>> $logFile
+
+    log "Adding Cluster extension"
+
+    $applianceId = (az arcappliance show --subscription $subscriptionId --resource-group $resourceGroupName --name $applianceName --query id -o tsv 2>> $logFile)
+    if (!$applianceId) {
+        throw "Appliance creation has failed."
+    }
+
+    # Waiting for the resource bridge to be in a running state
+    Do {
+        Write-Host "Waiting for the resource bridge to be in a running state, hold tight..."
+        Start-Sleep -Seconds 20
```

Comment:
```suggestion
        Write-Host "Waiting for the resource bridge to be in a running state, hold tight... (30 seconds loop)""
        Start-Sleep -Seconds 30
```

Reply:
It will reduce the number of lines you have in the screenshot and will provide some clarity. Also, you will need to take a new screenshot.